test/test_helper.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2014 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact Novell about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+# Set the paths
+SRC_PATH = File.expand_path("../../src", __FILE__)
+ENV["Y2DIR"] = SRC_PATH
+
+require "yast"
+
+if ENV["COVERAGE"]
+  require "simplecov"
+  SimpleCov.start
+
+  # for coverage we need to load all ruby files
+  Dir["#{SRC_PATH}/lib/**/*.rb"].each { |f| require_relative f }
+  Dir["#{SRC_PATH}/include/**/*.rb"].each { |f| require_relative f }
+  Dir["#{SRC_PATH}/modules/*.rb"].each { |f| require_relative f }
+  Dir["#{SRC_PATH}/clients/*.rb"].each { |f| require_relative f }
```

Comment:
Thanks, I'll adapt it. But I think your code is actually faulty. It reads "module" instead of "modules" and it lacks "include".

And yes, you are right, requiring the clients is playing with fire.